### DISCUSSION TITLE
highlight word regardless of if it is a whole word or a substring

### DISCUSSION THREAD

**Discussion by hoffor** [Chatterino/chatterino2#4422] [Q&A] [answered]
for example, i want the word 'hello' to trigger a highlight for 'ell'

according to what i've found on PCRE2 regex, using the word simply on its own should do exactly this, but it does not and i've read a comment in a chatterino issue also claiming a word on its own will only match the entire word

thank you

**Comment by Felanbird** [comment, marked as answer]
Are you sure you didn't read it backwards? It makes sense that `ell` matches `hello` but I don't see why `hello` should match `ell`

**Reply by hoffor** [reply]
that's what i said

**Reply by Felanbird** [reply]
![image](https://user-images.githubusercontent.com/41973452/222037791-bebd9014-a128-4ea2-a185-3fb320607190.png)

**Reply by hoffor** [reply]
doesn't highlight, dunno why

**Reply by Felanbird** [reply]
Did you check the `enable regex` box?

**Reply by hoffor** [reply]
derp. thank you, lol.
